Jenkinsfile

```diff
@@ -266,7 +248,7 @@ pipeline {
                     echo 'mkdocs stash does not exist'
                 }
 
-                archiveArtifacts artifacts: '**/dist/*.whl, **/docs/_build/html/**/*, **/external-docs/site/**/*', fingerprint: true
+                archiveArtifacts artifacts: '**/dist/*.whl, **/docs/_build/html/**/*, **/external-docs/site/**/*, **/test-results.json, **/tox-test-results-*.json, **/*results*.xml', fingerprint: true
```

Comment:
@GDonRanasinghe - I've added the test results files to the list of artifacts we should archive to ensure we can figure out what's gone wrong.

Reply:
@robons Many thanks, this will be highly useful.